patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -39,8 +39,8 @@
  
 +    /** Forge: Used to store custom data for each entity. */
 +    private NBTTagCompound customEntityData;
-+    public boolean captureDrops = false;
-+    public java.util.ArrayList<EntityItem> capturedDrops = new java.util.ArrayList<EntityItem>();
++    private List<EntityItem> capturedDrops = new java.util.ArrayList<EntityItem>();
```

Comment:
Does the order really matter here? We can change the base interface to collection